products/rhel8/profiles/stig.profile

```diff
@@ -641,7 +650,7 @@ selections:
     - account_disable_post_pw_expiration
 
     # RHEL-08-020270
-    - account_temp_expire_date
+    - account_emergency_expire_date
```

Comment:
What is the reason for this change? 

The two rules are very similar but the latest STIGs due use the temporary wording vs emergency. So I think that the existing mapping is fine.

Reply:
After re-reading more closely the STIG guidelines, I believe you are right ("Emergency accounts are not subject to manual removal or scheduled expiration requirements.").

However, that also means we should assign RHEL-08-020270 to account_temp_expire_date, and remove that same STIG reference in account_emergency_expire_date, right?
I will do that in the next version of this PR.

Reply:
> However, that also means we should assign RHEL-08-020270 to account_temp_expire_date, and remove that same STIG reference in account_emergency_expire_date, right? I will do that in the next version of this PR.

That sounds reasonable.